go/bin/spec

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -e
+
+bin/lint
+
+# runs go test on all subdirectories exluding the vendor folder
+# might cause issues with windows users though
```

Comment:
I was actually looking at morula and it looks like there is a never command to exclude certain directories. So I could just say `cd go && ../morula all --never=vendor bin/<command>` in the top bin folder.